build.gradle

```diff
@@ -98,6 +99,30 @@ subprojects {
   plugins.withType(BuildPlugin).whenPluginAdded {
     project.licenseFile = project.rootProject.file('licenses/APACHE-LICENSE-2.0.txt')
     project.noticeFile = project.rootProject.file('NOTICE.txt')
+
+    // Projects that should be formatted with Spotless are listed here, by
+    // project path. Once the number of formatted projects is greater than
+    // the number of unformatted projects, this can be switched to an
+    // exclude list, and eventualy removed completely.
+    def projectPathsToFormat = [
+      ':build-tools'
+    ]
+
+    if (projectPathsToFormat.contains(project.path)) {
+      project.apply plugin: "com.diffplug.gradle.spotless"
+
+      spotless {
+        java {
+          // Use Android Open Source Project settings (i.e. 4 space indents)
+          googleJavaFormat().aosp()
+
+          // See CONTRIBUTING.md for details of when to enabled this.
+          // paddedCell()
```

Comment:
Can we perhaps put this behind a flag? Say `-Dspotless.paddedCell` or something. I like to avoid toggles that involve commenting or uncommenting stuff because it's very easy for that stuff to be mistakenly committed to source control.